README.md

```diff
@@ -58,318 +35,57 @@ const tokenRes = await fetch('https://<ENVIRONMENT_AUTH_URL>/auth/realms/toonie/
-
-
-## Complete Self contained JS Snippet
-
-This snippet is performing what would normally be done serverside and clientside to just show a full implementation of the service.
-  
->**⚠️WARNING: NEVER perform any authentication towards Toonie's API on a Single Page Application or on any client-side app⚠️**
-
-The Full Example can be found [here](/samples/full_example/)
-
-`full_example.htm`
-```html
-<html>
-    <head>
-        <link rel="stylesheet" href="https://unpkg.com/@portittech/pay-with-toonie/dist/pay-with-toonie.dist.css">
-        <script src="https://unpkg.com/@portittech/pay-with-toonie/dist/pay-with-toonie.dist.js"></script>
-        <script src="full_script.js"></script>
-    </head>
-
-    <body>
-        <div>
-            <div id="toonie-button"></div>
-        </div>
-    </body>
-</html>
-```
-
-`full_example.js`
-```js
-const getTokenData = async () => {
-  // Auth to get token
-  // ATTENTION: MAKE SURE NOT TO INCLUDE THIS AUTHENTICATION SNIPPET IN YOUR CLIENTSIDE APPLICATION
-  // THIS HAS BEEN DONE FOR DEMONSTRATION PURPOSES ONLY!!!!
-  const tokenRes = await fetch(
-    "https://<ENVIRONMENT_AUTH_URL>/auth/realms/toonie/protocol/openid-connect/token",
-    {
-      method: "POST",
-      headers: {
-        "Content-Type": "application/x-www-form-urlencoded",
-      },
-      body: new URLSearchParams({
-        grant_type: "password",
-        client_id: "pay-with-toonie",
-        username: "customerusername",
-        password: "customerpassword",
-      }),
-    }
-  );
-
-  return await tokenRes.json();
-}
-
-/**
- * PAYMENT CREATION
- */
-
-const getPaymentData = async() => {
+// Create a payment session
+const createPaymentSession = async (amount, currency, reason) => {
   const tokenData = await getTokenData();
-  
-  //Create payment intent
-  const res  = await fetch('https://<ENVIRONMENT_API_URL>/offers/v1/payments', {
-    method: 'POST',
-    headers: {
-        Authorization: ` Bearer ${tokenData.access_token}`,
-        "content-type": "application/json",
-    },
-    body: JSON.stringify({
-        "amount": "0.05",
-        "reason": "Test Payment 01",
-        "destinationWalletId": "<MERCHANTWALLETID>",
-        "transactionCurrency": "EUR"
-    })
-  })
-
-    const data = await res.json()
-    
-    // Data to be consumed by the SDK
-    return { 
-      paymentSessionId: data.paymentSessionId,
-      otp: data.otp,
-      paymentShortReference: data.shortReference
-    }
-};
-
-/**
- * PAYMENT BY CARD CALLBACKS
- */
 
-const createCardPaymentIntent = async (paymentSessionId) => {
-  const tokenData = await getTokenData();
-
-  const res = await fetch('https://<ENVIRONMENT_API_URL>/acquiring/v1/card/custom', {
-    method: 'POST',
+  const res = await fetch("https://<ENVIRONMENT_API_URL>/acquiring/v1/payment", {
+    method: "POST",
     headers: {
-      Authorization: `Bearer ${tokenData.access_token}`,
+      Authorization: ` Bearer ${tokenData.access_token}`,
```

Comment:
```suggestion
      "Authorization": `Bearer ${tokenData.access_token}`,
```

Reply:
No need to add the `"`, it's js not json :)